Mathlib/CategoryTheory/Limits/Constructions/LimitsOfProductsAndEqualizers.lean

```diff
@@ -84,7 +84,7 @@ def buildIsLimit (t₁ : IsLimit c₁) (t₂ : IsLimit c₂) (hi : IsLimit i) :
     hi.hom_ext
       (i.equalizer_ext
         (t₁.hom_ext fun j => by
-          cases' j with j
+          obtain ⟨j⟩ := j
```

Comment:
More one-variable cases (no pun intended): perhaps worth discussing/was this discussed on zulip already?
I don't think these changes are *worse*, but I'm not convinced the current solution is ideal yet.

Reply:
Done.